plugins/trace_api_plugin/include/eosio/trace_api/request_handler.hpp

```diff
@@ -42,12 +43,15 @@ namespace eosio::trace_api {
          }
 
          yield();
-
-         auto data_handler = [this](const action_trace_v0& action, const yield_function& yield) -> fc::variant {
-            return data_handler_provider.process_data(action, yield);
+         
+         auto data_handler_v0 = [this](const action_trace_v0& action, const yield_function& yield) -> std::tuple<fc::variant, fc::optional<fc::variant>> {
+            return data_handler_provider.template process_data<action_trace_v0>(action, yield);
```

Comment:
Template deduction can determine the type from just calling the function.  There is no need to have the `template` and `<action_trace_v0>`.

Reply:
good idea